invenio_files_rest/views.py

```diff
@@ -594,6 +594,17 @@ def delete_object(self, bucket, obj, version_id):
             db.session.commit()
             if obj.file_id:
                 remove_file_data.delay(str(obj.file_id))
+            # Set newest object as head
+            if obj.is_head:
```

Comment:
Do you mind to add a couple of tests?
1) happy path, you have 2 versions, delete the first, the second should have `is_head == True`
2) you have only 1 version, delete the first, then the bucket should be empty

Reply:
Good suggestion! I added 1st test. For 2nd, I am not sure if it is needed, as far as I understood it is already there - https://github.com/inveniosoftware/invenio-files-rest/blob/7f9014035d8aac5845aa2fd96c134891e4c2c96b/tests/test_views_bucket.py#L130